firewallrule.go

```diff
@@ -43,40 +42,40 @@ type FirewallRules struct {
 
 //ListFirewallRules lists all firewall rules
 func (c *Client) ListFirewallRules(dcID string, serverID string, nicID string) (*FirewallRules, error) {
-	url := fwruleColPath(dcID, serverID, nicID) + `?depth=` + c.client.depth + `&pretty=` + strconv.FormatBool(c.client.pretty)
+	url := fwruleColPath(dcID, serverID, nicID)
 	ret := &FirewallRules{}
-	err := c.client.Get(url, ret, http.StatusOK)
+	err := c.Get(url, ret, http.StatusOK)
```

Comment:
Why aren't you using the `OK`/`Acc` helper methods?

Reply:
I did not want to fix all calls. Can be done in later PR